### DISCUSSION TITLE
Storage size for kafka and zookeeper pods in production environment

### DISCUSSION THREAD

**Discussion by shreyasarani23** [strimzi/strimzi-kafka-operator#4934] [General]
What would be the ideal storage size for kafka and zookeeper pods if I am having 3 replicas of kafka and zookeeper pods considering the fact it will be running in production environment?

I know the fact it depends on individual's workload like number of events that are getting written into topics, depends on producers and consumers etc. I am asking your opinion what would be the ideal size considering for production environment?

**Comment by sknot-rh** [comment]
Hi. As you say, it is heavily dependent on the scenarios you will have in your cluster. I would point you to the [calculator](https://eventsizer.io/simple) to get a better idea from the data you can provide.

**Reply by shreyasarani23** [reply]
Thanks @sknot-rh
